PLEP-0000.rst

```diff
@@ -2,81 +2,116 @@
-+--------+---------------------------------------------------------------------------+---------------------------+---------------+---------------------------------------------+
-| 5      | `PlasmaPy Versioning and Releases                                         | Nicholas A. Murphy and    | process       | `10.5281/zenodo.1451974                     |
-|        | <https://github.com/PlasmaPy/PlasmaPy-PLEPs/blob/master/PLEP-0005.rst>`__ | Stuart J. Mumford         |               | <https://doi.org/10.5281/zenodo.1451974>`__ |
-+--------+---------------------------------------------------------------------------+---------------------------+---------------+---------------------------------------------+
-| 6      | `A New General-Purpose Plasma Object                                      | Andrew J. Leonard         | standard      | `10.5281/zenodo.1460976                     |
-|        | <https://github.com/PlasmaPy/PlasmaPy-PLEPs/blob/master/PLEP-0006.rst>`__ |                           |               | <https://doi.org/10.5281/zenodo.1460976>`__ |
-+--------+---------------------------------------------------------------------------+---------------------------+---------------+---------------------------------------------+
++--------+--------------------------------------------------------------+----------------------+------------+---------------------------+
+| number | title                                                        | author(s)            | type       | DOI                       |
++========+==============================================================+======================+============+===========================+
+| 1      | `Purpose and Guidelines for PlasmaPy Enhancement Proposals`_ | Nicholas A. Murphy   | process    | `10.5281/zenodo.1435976`_ |
++--------+--------------------------------------------------------------+----------------------+------------+---------------------------+
+| 2      | `PlasmaPy Governance`_                                       | Nicholas A. Murphy   | process    | `10.5281/zenodo.1435980`_ |
++--------+--------------------------------------------------------------+----------------------+------------+---------------------------+
+| 4      | `Licensing of PlasmaPy Repositories`_                        | Nicholas A. Murphy   | process    | `10.5281/zenodo.1435990`_ |
++--------+--------------------------------------------------------------+----------------------+------------+---------------------------+
+| 6      | `A New General-Purpose Plasma Object`_                       | Andrew J. Leonard    | standard   | `10.5281/zenodo.1460976`_ |
++--------+--------------------------------------------------------------+----------------------+------------+---------------------------+
+| 7      | `Structure of Top-Level Sub-Packages`_                       | Erik T. Everson,     | standard   |                           |
```

Comment:
This is missing its DOI.

Reply:
This hasn't been uploaded to Zenodo, so it does not yet have a DOI.

Reply:
I lied. It does have one, which I'll add.

 - [ ] Do ↑